include/swift/AST/SourceFile.h

```diff
@@ -63,11 +67,16 @@ class SourceFile final : public FileUnit {
   struct ImportedModuleDesc {
     ModuleDecl::ImportedModule module;
     ImportOptions importOptions;
+
+    // Filename for a @_private import.
     StringRef filename;
 
+    // Names of explicitly imported SPIs.
+    ArrayRef<Identifier> spis;
```

Comment:
can we rename this to `spiKinds`? `spis` sounds like these are the decl identifiers of decls marked as SPI.